webpack.config.babel.js

```diff
@@ -134,9 +137,9 @@ module.exports = ({ stats = false } = {}) => ({
       mode: 'webapp', // optional can be 'webapp' or 'light' - 'webapp' by default
       devMode: 'webapp', // optional can be 'webapp' or 'light' - 'light' by default
       favicons: {
-        appName: 'Mesa',
-        appDescription: 'Mesa',
-        developerName: 'Mesa',
+        appName: appName,
+        appDescription: appName,
+        developerName: appName,
```

Comment:
I was going for a higher granularity, but this is fine because this is all just for the favicon I guess?

Reply:
Speaking of favicon, a more meaningful config var would be `favicon.logo` for custom icon path

Reply:
Yep, I wouldn't overcomplicate this allowsing too much granularity

For the logo, I agree, I think indeed we should allow to set/override it, OR use a naming convention, or both `custom/favicon.png`, not sure if it's good or bad :)